internal/pkg/template/mappers.go

```diff
@@ -67,6 +72,18 @@ func MappersFor(s *state.State, d dependencies, ctx Context) (mapper.Mappers, er
 		replacevalues.NewMapper(s, replacements),
 	}
 
+	// Add metadata on "template preview" operation
+	if c, ok := ctx.(previewContext); ok {
+		if _, isUseContext := ctx.(useContext); !isUseContext {
```

Comment:
Can we put this into single `if` condition?

Reply:
For me, it's more readable in this format.

Reply:
For me this is even more readable :smile: